src/main/resources/META-INF/services/javax.cache.spi.CachingProvider

```diff
@@ -0,0 +1 @@
+com.github.mizool.jediscache.JedisCachingProvider
```

Comment:
Use the [meta inf services annotation processor](http://metainf-services.kohsuke.org/) with the `@MetaInfServices` annotation on the `JedisCachingProvider` instead.
```xml
<dependency>
    <groupId>org.kohsuke.metainf-services</groupId>
    <artifactId>metainf-services</artifactId>
    <version>1.7</version>
</dependency>
```